static/app/components/profiling/profilingTransactionHovercard.tsx

```diff
@@ -104,6 +117,19 @@ export function ProfilingTransactionHovercardBody({
     });
   };
 
+  const handleLinkToFlamechartAnalyticEvent = () => {
+    trackAdvancedAnalyticsEvent('profiling_views.go_to_flamegraph', {
+      organization,
+      source: 'transaction_hovercard',
+    });
+  };
```

Comment:
Do we want to use different sources if they're clicking the latest vs the slowest?

Reply:
hmmm .. i like that idea

Reply:
✅